spotify/models/user.py

```diff
@@ -78,6 +143,23 @@ def __init__(self, client: "spotify.Client", data: dict, **kwargs):
         else:
             self.library = Library(client, self)
 
+        self.token_info = token_info or TokenInfo()
+
+        self._cache_path: Optional[Path] = kwargs.pop("cache_path", None)
+        if self._cache_path and not self._cache_path.exists():  # make sure the file and folders exist
```

Comment:
No, I'm not keen on this block implicitly creating files and directories. Instead i believe that we should raise `FileNotFound` when a cache_path is specified but invalid